### PR TITLE
docs: grammar fix and broken md link in docs

### PR DESCRIPTION
**Pull request by coltraneyan**
- added some minor grammar fixes I noticed as I was reading the docs
- updated a typo causing a broken markdown link in the starters

### CODE REVIEW

File: content/tutorial/index.md [hunk 1]
```diff
@@ -15,17 +16,19 @@ If you are someone who prefers to stare at the code till it explains itself, you
...
   - [Add JavaScript to Components](#add-javascript-to-components)
 - [Dynamic Page Generation](#dynamic-page-generation)
+
 ---
 
 ## What is Abell?
 
 Abell is a JavaScript-based Static-Site-Generator to help you build fast and scalable static websites. It is similar to tools like Jekyll, Eleventy, Hugo, and Hexo.
 
-Abell files are like HTML files except you can write JavaScript in double curly brackets which is executed to build you a static page.
```
Comment (saurabhdaware) [inline content/tutorial/index.md:24]: "build you a static page" is correct I believe. It is building a static page for you right?

File: content/tutorial/index.md [hunk 2]
```diff
@@ -92,10 +100,10 @@ Let's start by changing **Abell Example** with your name.
 module.exports = {
   // ..
   globalMeta: {
-    siteTitle: 'Saurabh', // replace with your name
-  }
+    siteTitle: "Saurabh", // replace with your name
+  },
   //...
-}
```
Comment (saurabhdaware) [inline content/tutorial/index.md]: These changes are formatting changes right? maybe you can revert these. I should've added prettier. Will do that in new version.

**Review by saurabhdaware — CHANGES_REQUESTED** [review]
Heya! Thanks! Added some minor comments

File: content/tutorial/index.md (hunk 2 above)
Comment (coltraneyan) [reply content/tutorial/index.md]: Thanks for catching that, I reverted this.

File: content/tutorial/index.md (hunk 1 above)
Comment (coltraneyan) [reply content/tutorial/index.md:24]: Reverted back to original

**Review by saurabhdaware — APPROVED** [review]
Awesome! Thank you so much for contributing 🎉

**Pull request merged**